src/containers/NewClaim.tsx

```diff
@@ -131,13 +130,12 @@ class NewClaim extends React.Component<Props, State> {
             status: ClaimStatus.AttestationPending,
             contents: requestForAttestation.claim.contents,
             requestTimestamp: Date.now(),
-            data: requestForAttestation,
+            req4Att: requestForAttestation,
           })
-          const claimerIdentity = fromStoredIdentity(identityFromStore)
           await sendRequestForAttestation(
             requestForAttestation,
-            claimerIdentity,
-            attesterPublicIdentity
+            identityFromStore,
```

Comment:
my point from the `publicIdenetityFromStore` we don't need the `publicIdentityFromStore` if we have `identityFromStore`